src/pages/resources/resources.ts

```diff
@@ -109,18 +111,45 @@ export const powerupPrice = derived(
     }
 )
 
-export const stakingPrice = derived(
+// price per kb
+export const netPowerupPrice = derived(
+    [sampleUsage, statePowerUp, info],
+    ([$sampleUsage, $statePowerUp, $info]) => {
+        if ($sampleUsage && $statePowerUp) {
+            return Asset.from(
+                $statePowerUp.net.price_per_kb($sampleUsage, 1, $info),
+                '4,EOS'
```

Comment:
We can't assume that this will be `EOS`, we need to use `$activeBlockchain.coreTokenSymbol` as we did [here](https://github.com/greymass/unicove/pull/211/files#diff-df9cf161ab2cb650e82e72adc950f1945a437e89fe5ce8b728a9b32c420af4beL123).